user.go

```diff
@@ -112,16 +112,17 @@ func (user *User) Teams(client *Client, variables *PayloadVariables) (*TeamIdCon
 	return &q.Account.User.Teams, nil
 }
 
-func (client *Client) InviteUser(email string, input UserInput) (*User, error) {
+func (client *Client) InviteUser(email string, input UserInput, forceSendInvite *bool) (*User, error) {
```

Comment:
```suggestion
func (client *Client) InviteUser(email string, input UserInput, sendInvite bool) (*User, error) {
```

Does it need to be a bool pointer?  Since we are now requiring the end user to specify during invite time we can just pass `true` or `false` - that would be nicer to the end user instead of having to `ol.RefOf(true)`

Reply:
updated to be a bool